pym/bob/cmds/build/project.py

```diff
@@ -107,7 +107,11 @@ def _downloadArg(arg):
             parser.error("--jobs argument must be greater than zero!")
         extra.extend(['-j', str(args.jobs)])
 
-    package = packages.walkPackagePath(args.package)
+    if generator.get('query'):
+        package = packages.queryPackagePath(args.package)
```

Comment:
I guess a simple test for such a new generator should be added.

Reply:
> I'm wondering how much effort it would be to support queries in the existing generators...

I'm not sure what the expected result is for the existing generators? ATM we generate one project in `projects` for the given package. If a query matches multiple packages we would generate one project for each match?